docker-compose.yml

```diff
@@ -33,6 +33,8 @@ services:
       ZOOKEEPER_SSL_TRUSTSTORE_LOCATION: /etc/kafka/secrets/kafka.zookeeper.truststore.jks
       ZOOKEEPER_SSL_TRUSTSTORE_PASSWORD: confluent
       ZOOKEEPER_SSL_TRUSTSTORE_TYPE: JKS
+      ZOOKEEPER_SSL_ENABLED_PROTOCOLS: "TLSv1.2"
```

Comment:
If ZK does not support v1.3, why is it required to configure v1.2?  --> Can this line be removed?

Reply:
For cp-demo yes this can be removed.  I'm trying to recall if there are any JDKs or other factors that for ZooKeeper allow TLS 1.1 or earlier, but if there are they don't apply to cp-demo ... so this setting is only there for completeness, but would block TLS 1.3 should it ever be supported, so will remove.